dds/DCPS/transport/multicast/MulticastSendStrategy.cpp

```diff
@@ -16,7 +16,7 @@ namespace OpenDDS {
 namespace DCPS {
 
 MulticastSendStrategy::MulticastSendStrategy(MulticastDataLink* link)
-  : TransportSendStrategy(0, link->impl(),
+  : TransportSendStrategy(0, *link->impl(),
```

Comment:
Is this not setting up the same kind of problem?

Reply:
Yes, probably, but I was hesitant to dig into the strategies as well. I guess I can add it.